.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+
+indent_style = space
```

Comment:
I prefer spaces :) But I am ok with that - updated code there, changed eslint rules for tabs